src/main/java/com/example/icecreamup/dto/OrderDTO.java

```diff
@@ -17,7 +17,7 @@
 	private String email;
 	private Date timeStamp;
 	private String comments;
-	private List<OrderItemDTO> orderItems;
+	private List<OrderProductsDTO> orderProducts;
```

Comment:
OrderItemDTO does not make sense, OrderItems are products so for clarity refer to them as such